src/celutil/debug.cpp

```diff
@@ -10,38 +10,30 @@
 #ifdef _MSC_VER
 #include <windows.h>
 #endif
-#include <cstdio>
-#include <cstdarg>
-#include <config.h>
+#include <iostream>
 #include <fmt/printf.h>
 
 static int debugVerbosity = 0;
 
 #if defined(DEBUG) || defined(_DEBUG)
-void DebugPrint(int level, const char *format, ...)
+template <typename... T>
+void DebugPrint(int level, const char *format, const T & ... args)
```

Comment:
the template function should be declared and implemented in header?, this is not helping with the error of passing an object as varargs

Reply:
correct, i just checked g++ debug.cpp, will fix it later